README.md

```diff
@@ -273,10 +273,14 @@ OPTIONS
 
   -s, --tls
       Enable TLS encryption.
-                           Note, that this option is turned on by default for kubernetes infrastructure.
-                           If it is needed to provide own certificate, 'che-tls' secret with TLS certificate must be 
-      created in the configured namespace. Otherwise, it will be automatically generated.
+                           Note, this option is turned on by default.
+                           For Kubernetes infrastructure, it is required to provide own certificate: 'che-tls' secret with 
```

Comment:
does not kubectl generate it if it's missing?

Reply:
I don't understand your question completely, could you clarify please?
If you are talking about helm installer - then yes, it will generate the secret and set self-signed flag, but these checks are added to operator tasks list.

Reply:
But it's help output of server:start command, right?
Not helm or operator installer specified by user at this stage...
As an user I would read it and expect that secret must be created on K8s before I run `chectl server:start --installer=helm --platform=kubernetes`. Sorry if I missed something or is still unclear.

Reply:
Agree, it is worth to add a note about Helm. Thank you for pointing out.